webpack/envs/clientDevelopmentConfig.js

```diff
@@ -25,9 +21,6 @@ if (!env.onCi && !fs.existsSync(cacheDirectory)) {
 }
 
 export const clientDevelopmentConfig = {
-  devServer: {
-    hot: true,
-  },
```

Comment:
Let me pull this PR down and see if this still works as expected

Reply:
Or read the docs, all that does is add the plugin to the config.

Reply:
It looks like there might be a bug with the dev server, or some kind of configuration problem between `master` and this PR:

Repo steps: 
- boot app
- navigate to `/collect` 
- make a change to the [title located here](https://github.com/artsy/force/blob/master/src/v2/Apps/Collect/Routes/Collect/index.tsx#L86)
- hit save, triggering hot reloading

In the console, i see four separate instances of the HMR middleware firing, which takes on average 7 seconds to refresh the page: 

<img width="477" alt="Screen Shot 2021-03-10 at 2 23 34 PM" src="https://user-images.githubusercontent.com/236943/110707202-d05ccc80-81ad-11eb-8ebb-1c64ea39b665.png">

Checking out `master` and following the same steps above, I'm getting one instance of the server triggering a change, and the complete reload cycle takes ~1.5sec

<img width="499" alt="Screen Shot 2021-03-10 at 2 31 44 PM" src="https://user-images.githubusercontent.com/236943/110707298-f5513f80-81ad-11eb-9c3b-991bcfdec77c.png">